web/templates/note/form.html.eex

```diff
@@ -0,0 +1,9 @@
+<%= form_for @changeset, @action, fn f -> %>
+<div>
+  <%= text_input f, :title , placeholder: "Title" %>
+</div>
+<div>
+  <%= text_input f, :description , placeholder: "Enter note here..." %>
```

Comment:
I'm going to comment on this now, even though you won't be able to test it until you have an update/create, but what happens if the form fails to submit? You'd probably want to display errors next to the field. Check out `error_tag` in `web/views/error_helpers.ex`.